README.md

```diff
@@ -4,6 +4,7 @@
 
 - **Version 2**
 
+  - 2.0.6: Added optional icon with default icon to both popup buttons and added a handleClick function to the DownloadCard;
```

Comment:
Since you've really updated two separate components, I think it would be best to give them their own separate bullet point, like so:

- 2.0.6
  - Changes made in component x;
  - Changed made in component y;